backend/app/api/main.go

```diff
@@ -72,6 +75,45 @@ func run(cfg *config.Config) error {
 	app := new(cfg)
 	app.setupLogger()
 
+	if cfg.Options.AllowAnalytics {
+		type analyticsData struct {
+			Domain string                 `json:"domain"`
+			Name   string                 `json:"name"`
+			Url    string                 `json:"url"`
+			Props  map[string]interface{} `json:"props"`
+		}
+		hostData, _ := host.Info()
+		analytics := analyticsData{
+			Domain: "homebox.software",
+			Url:    "https://homebox.software",
+			Name:   "stats",
+			Props: map[string]interface{}{
+				"os":               hostData.OS,
+				"platform":         hostData.Platform,
+				"platform_family":  hostData.PlatformFamily,
+				"platform_version": hostData.PlatformVersion,
+				"kernel_arch":      hostData.KernelArch,
+			},
+		}
+		jsonBody, err := json.Marshal(analytics)
+		if err != nil {
+			log.Error().Err(err).Msg("failed to marshal analytics data")
+		}
+		bodyReader := bytes.NewReader(jsonBody)
+		req, err := http.NewRequest("POST", "https://a.sysadmins.zone/api/event", bodyReader)
+		req.Header.Set("Content-Type", "application/json")
+		req.Header.Set("User-Agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36 OPR/71.0.3770.284")
```

Comment:
Need to set a real user agent here instead of just browser user agent (this was just for testing).